### ISSUE TITLE
Exit as failure if warnings are present?

### ISSUE THREAD

**Issue by cprodescu** [facebook/flow#5009]
In 0.54.0 ([commit](https://github.com/facebook/flow/commit/c62d5ebac2ef7881192b760b023ce54bb57140e6)) unused suppressions (`$FlowFixMe`) became warnings. My use-case for suppressions is to assert flow errors where those should exist (in `__flowtests__` files). These tests give me confidence in my local flow-typed interfaces when upgrading flow (and vice-versa).

Is there a way to get failure exit code if warnings are present? (either via `.flowconfig` or command-line arguments)?
Alternatively, do you have other recommendations?

Thank you!

**Comment by villesau** [comment]
I think this is pretty serious problem for flow-typed: https://github.com/flowtype/flow-typed/issues/1385

**Comment by cprodescu** [comment]
FWIW, our current workaround is to enable warnings in flow config (via `include_warnings=true`) and run a slightly modified script in CI:
```
[[ $(flow check) == 'Found 0 errors' ]]
```
instead of just
```
flow check
```

**Comment by villesau** [comment]
Similar solution was introduced for flow-typed.

**Comment by cprodescu** [comment]
This was resolved in 0.66.x via https://github.com/facebook/flow/commit/d6682b2422515c294710e4a1a84cc01b316806b4

To get flow to exit with failure if errors are present, use
```
flow --max-warnings 0
```

Thank you @nmote!

**Comment by NameFILIP** [comment]
Is it possible to specify `max_warnings` in the `.flowconfig` file?
